js/modules/treemap.src.js

```diff
@@ -609,27 +657,46 @@ seriesType(
                 this.axisTypes = colorSeriesMixin.axisTypes;
             }
 
-            Series.prototype.init.call(series, chart, options);
+            addEvent(series, 'setOptions', function (event) {
+                var options = event.userOptions;
 
-            options = series.options;
+                // Test if deprecated option is set.
+                if (defined(options.allowDrillToNode)) {
+                    error(
+                        'WARNING: plotOptions.treemap.allowDrillToNode has ' +
+                        'been renamed to plotOptions.treemap.' +
+                        'allowTraversingTree, and will be removed in the ' +
+                        'next major version.'
```

Comment:
1. Export text of error to number of error (like error 19/25).
2. Add chart context (third param of error).

Example:
```
H.error(18, true, chart);
```

It will allow us to keep full compatibility with `debug module`.

Reply:
I had a chat with Torstein about this, and we do not have a practice of warning of deprecations in the console. So ended up removing the warning.

If we should use the error system, then it would not only have to work with the debug module, but also it has to be flexible enough to specify which deprecated option is used, which is not possible when using error numbers.
Something to keep in mind for the future.